shared/constants/push.js

```diff
@@ -13,10 +13,13 @@ export type PushNotification = {
   payload?: {
     userInteraction: boolean,
     convID?: string, // Android variant
+    type?: string,
```

Comment:
Yeah we could also put this in AVDL, probably should have done that from the beginning. It's probably cool to just put the type in manually, and I can ticket moving the payload types into AVDL so we can just get it for free.